jwst/pipeline/calwebb_spec3.py

```diff
@@ -87,6 +87,7 @@ def process(self, input_data):
         self.resample_spec.suffix = "s2d"
         self.resample_spec.save_results = self.save_results
         self.cube_build.suffix = "s3d"
+        self.cube_build.pipeline = (3,)
```

Comment:
Here too - just set an int.
```suggestion
        self.cube_build.pipeline = 3
```